test/runtests.jl

```diff
@@ -98,10 +98,10 @@ const LQOI = LinQuadOptInterface
         MOIT.lin2vtest(solver, config)
         set_lin2test_solutions!(solver)
         MOIT.lin2ftest(solver, config)
-        set_lin3test_solutions!(solver)
-        MOIT.lin3test(solver, config)
-        set_lin3test_solutions!(solver)
-        MOIT.lin4test(solver, config)
+        # set_lin3test_solutions!(solver)
```

Comment:
I commented these to make sure the other tests are passing. We should solve this https://github.com/JuliaOpt/LinQuadOptInterface.jl/pull/64#issuecomment-418730742 and uncomment them before we merge